src/PageRouter.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {
+  Route,
+  Routes,
+  useLocation,
+  useNavigate,
+} from 'react-router-dom';
+
+import { App } from './App';
+import { HomePage } from './components/HomePage';
+import { PeoplePage } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage';
+
+export const PageRouter = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  React.useEffect(() => {
+    if (location.pathname === '/home') {
```

Comment:
The useEffect redirects /home to / which might not be desired. If /home should be a valid path, this redirection should be removed. Alternatively, if /home should not be a valid path, then this could be better handled with a ```<Redirect>``` component in the ```<Routes>```.